ingestion-beam/src/main/java/com/mozilla/telemetry/schemas/JSONSchemaStore.java

```diff
@@ -18,11 +19,21 @@
 
   /** Returns a SchemaStore based on the contents of the archive at schemasLocation. */
   public static JSONSchemaStore of(ValueProvider<String> schemasLocation) {
-    return new JSONSchemaStore(schemasLocation);
+    return JSONSchemaStore.of(schemasLocation, null);
   }
 
-  protected JSONSchemaStore(ValueProvider<String> schemasLocation) {
-    super(schemasLocation);
+  /**
+   * Returns a SchemaStore based on the contents of the archive at schemasLocation
+   * with additional schemas aliased according to configuration.
+   */
+  public static JSONSchemaStore of(ValueProvider<String> schemasLocation,
+      @Nullable ValueProvider<String> aliasingConfigurationLocation) {
```

Comment:
I have a slight preference for simplifying this by removing the single argument constructor and making this a required field rather than nullable, since the real use in the Decoder will always be passing in both value providers, even if the produced value ends up being empty.